src/lib.rs

```diff
@@ -105,13 +106,35 @@ impl Deref for RedactedString {
     }
 }
 
+/// Authentication method for Deepgram API requests.
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub enum AuthMethod {
```

Comment:
This should also be `pub(crate)`.  It does not need to be exposed to the end user.